dataland-frontend/tests/e2e/specs/admin-tools/AdminConsoleSecurity.ts

```diff
@@ -28,6 +28,6 @@ describe("As a user I expect the admin console only to be reachable using admin-
       .get("#kc-login")
       .should("exist")
       .click();
-    cy.get("h1").should("exist").should("contain", "Master realm");
+    cy.get("h1").should("exist").should("contain", "master realm");
```

Comment:
Ist das einfach nur eine Änderung aufgrund der neuen keycloak Version?
Also schreiben die das einfach in lowercase ab jetzt in ihrer UI?

Reply:
Das wäre jetzt zumindest meine Vermutung